include/ogdf/internal/decomposition/TricComp.h

```diff
@@ -49,14 +49,30 @@ namespace ogdf {
 class TricComp
 {
 public:
-	//! construction
+	/**
+	 * Divides G into triconnected components.
+	 * \param G graph
+	 */
 	TricComp(const Graph &G);
 
+	/**
+	 * Tests G for triconnectivity.
+	 * \param G graph
+	 * \param isTric true if G is triconnected, false otherwise.
+	 * \param s1 first vertex of separation pair if G is biconnected, cut vertex of G if G is not biconnected, nullptr if G is not connected.
+	 * \param s2 second vertex of separation pair if G is biconnected, nullptr otherwise.
+	 */
 	TricComp(const Graph &G, bool &isTric, node &s1, node &s2);
 
+	/**
+	 * Copy constructor.
+	 */
 	TricComp(const TricComp &) = delete;
 
-	//! destruction
+	/**
+	 * Destructor.
+	 * Deletes GraphCopy.
```

Comment:
> mentioned in TricComp.cpp

That's okay as it might be relevant to the internals of the class.

> Language syntax already tells us it's a destructor

Totally agree, no need to have this comment in the docs. :)